ui-tests/cypress/support/index.d.ts

```diff
@@ -29,12 +29,12 @@ import UserPreferencesController       from './controllers/restAPI/UserPreferenc
 import UsersController                 from './controllers/restAPI/UsersController';
 import WidgetsController               from './controllers/restAPI/WidgetsController';
 
-import CategoriesAdminConsoleController        from './controllers/cmsActions/CategoriesAdminConsoleController';
-import ContentsAdminConsoleController          from './controllers/cmsActions/ContentsAdminConsoleController';
-import ContentTemplatesAdminConsoleController  from './controllers/cmsActions/ContentTemplatesAdminConsoleController';
-import VersioningAdminConsoleController        from './controllers/cmsActions/VersioningAdminConsoleController';
-import ContentTypesAdminConsoleController      from './controllers/cmsActions/ContentTypesAdminConsoleController';
-import ContentTypesJacmsAdminConsoleController from './controllers/cmsActions/ContentTypesJacmsAdminConsoleController';
```

Comment:
Have you removed the two _ContentTypes_ controllers?

Reply:
sorry, when I did the rebase I accidentally removed the two controllers. 
I only did the ammend for controllersEndpoints, without reinserting them in the imports. 
I will see to it immediately.